docs/docs/migration.md

```diff
@@ -72,20 +91,18 @@ resources:
     type: com.sap.xs.uaa
     parameters:
       path: ./xs-security.json
-
 ```
 
-And your `xs-security.json` file contains a property which value should be resolved during the MTA archive deployment:
+and your `xs-security.json` file contains a property which value should be resolved during the MTA archive deployment:
```

Comment:
```suggestion
and your `xs-security.json` file contains a property whose value should be resolved during the MTA archive deployment:
```
(at least I think it should be 'whose', you should ask Paola)

Reply:
I will double-check with Paola.